### PR TITLE
[fp8 rowwise] Simplify epilogue visitor tree via common blocks

### PR DESCRIPTION
**Pull request by lw**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #134227
* #134226
* #134225
* #134224
* __->__ #134223

### CODE REVIEW

File: aten/src/ATen/native/cuda/RowwiseScaledMM.cu [hunk 1]
```diff
@@ -74,6 +74,32 @@ using DtypeAccum = float;
 using DtypeEpilogue = float;
 using DtypeOutput = cutlass::bfloat16_t;
 
+template <typename T>
+struct identity {
```
Comment (drisspg) [inline aten/src/ATen/native/cuda/RowwiseScaledMM.cu:78]: noob q: is it better to reuse the ones defined in cutlass: https://github.com/NVIDIA/cutlass/blob/f7b19de32c5d1f3cedfc735c2849f12b537522ee/include/cutlass/epilogue/thread/activation.h#L57
or 
https://github.com/NVIDIA/cutlass/blob/f7b19de32c5d1f3cedfc735c2849f12b537522ee/include/cute/algorithm/functional.hpp#L46
?

File: aten/src/ATen/native/cuda/RowwiseScaledMM.cu [hunk 2]
```diff
@@ -156,54 +182,29 @@ void f8f8bf16_rowwise_impl(
...
-      TileShape,
-      DtypeScale,
-      cute::Stride<cute::Int<0>, cute::Int<1>, cute::Int<0>>>;
-
-  using Bias = cutlass::epilogue::fusion::Sm90RowBroadcast<
-      PONG ? 2 : 1,
-      TileShape,
-      DtypeBias,
-      cute::Stride<cute::Int<0>, cute::Int<1>, cute::Int<0>>>;
+  constexpr int ColBcastStages = 0;
```
Comment (drisspg) [inline aten/src/ATen/native/cuda/RowwiseScaledMM.cu:185]: should this be ColA
?

**Review by drisspg — APPROVED** [review]

File: aten/src/ATen/native/cuda/RowwiseScaledMM.cu (hunk 1 above)
Comment (lw) [reply aten/src/ATen/native/cuda/RowwiseScaledMM.cu:78]: Ah, thanks, I was looking for it here and couldn't find it! https://github.com/NVIDIA/cutlass/blob/f7b19de32c5d1f3cedfc735c2849f12b537522ee/include/cutlass/functional.h

File: aten/src/ATen/native/cuda/RowwiseScaledMM.cu (hunk 2 above)
Comment (lw) [reply aten/src/ATen/native/cuda/RowwiseScaledMM.cu:185]: I was using `Bcast` as a shorthand for `Broadcast` but let me expand it to avoid confusion.

Comment (drisspg) [reply aten/src/ATen/native/cuda/RowwiseScaledMM.cu:185]: ohhh my bad I though it was Cols B tensor

**Pull request closed without merging**